sarracen/interpolate.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+
+
+def interpolate2D(data, x, y, target, kernel, xmin, ymin, pixwidthx, pixwidthy, pixcountx, pixcounty):
```

Comment:
Possibly a good idea to set sensible defaults for the pixcounts. Try pixcountx=480 and pixcounty=480. This assumes the particles are within a square domain, but maybe that's ok for now. We can refactor if/when it becomes a problem.